Classes/Issues/IssuesViewController.swift

```diff
@@ -386,7 +389,14 @@ IssueCommentSectionControllerDelegate {
             objects.append(rootComment)
         }
         
-        objects += current.timelineViewModels
+        objects += current.timelineViewModels.filter({ (model) -> Bool in
```

Comment:
Not sure if this matters, but I *think* `append` might be more performant here? Thinking about the concatenation of strings. But like I said, not sure. 😛

Reply:
🤷‍♂️  Didn't change it versus what was already there, maybe one to explore